page_objects/searchResultPage.js

```diff
@@ -0,0 +1,53 @@
+class SearchResultPage {
+  constructor() {
+    this.searchResults = element.all(by.css(".search-result__item"));
+    this.form = element(by.css(".form-component__description div"));
```

Comment:
This selector is not in the search result page, It is in the job description page.

Reply:
deleted